### PR TITLE
SOLARCH-278 updating bundle instructions to pdk

### PR DESCRIPTION
**Pull request by davidsandilands**
Since the setup uses pdk it would reduce complexity to use bundle via pdk for running and testing plans.

### CODE REVIEW

**Review by lucywyman — APPROVED** [review]
Thanks for this update!

**Review by hestonhoffman — APPROVED** [review]
🙌

**Comment by lucywyman** [comment]
Ah, ok, I see now why we didn't initially include this command: it's experimental https://github.com/puppetlabs/pdk#experimental-features I'd like to wait until it's not an experimental change to merge this so that users don't run into unhandled errors or anything. I'll keep an eye on the feature though and I think this change would be great once the command is fully baked in!

**Comment by jpogran** [comment]
`pdk bundle` is marked as experimental to indicate that anything executed that way is outside of the 'easy mode' path of the pdk command structure. Think 'advanced mode' instead of 'this does what I need for me without me typing things'.

File: documentation/testing_plans.md
Comment (jpogran) [inline documentation/testing_plans.md:38]: I haven't tested with bolt plans, but if you go the route of using the PDK then this should be the workflow:

```
> pdk new module foo
> cd foo
> # add tests
> pdk validate  # does a gem install using the PDK env, runs spec_prep, etc
> pdk test unit  # runs all tests, ensuring gem install, etc. equivalent of all the manual rake spec commands
```

Comment (davidsandilands) [reply documentation/testing_plans.md:38]: @jpogran you are absolutely right. I'll update the pull to use full pdk instructions. I wasn't looking at big picture. Thanks :-)

**Review by beechtom — APPROVED** [review]

**Pull request merged**
